### ISSUE TITLE
Support different Sell and Buy prices

### ISSUE THREAD

**Issue by JanReich** [Bounser/Nascraft#11]
At the moment we have decided to rework our economy system. We have found your plugin and it is really awesome. For our gamemode we need a separate buy and sell value. For example oak wood is buyable by players between 50 and 100 coins and sellable between 1 and 5 coins. 
The idea was that the pricedifferenze in percent is the same between the buy and the sell option but there are different prices. For example oak wood is now 10% cheaper. The buying value was 50 coins now it is 45 coins and the selling value was 2.5 coins and now it is 2.25 coins. 
This feature would be realy awesome and then we would be able to use this plugin at our server.

**Comment by JanReich** [comment]
Okay it worked with the different buy/sell actions. 

Another point it is possible to open a category from a shop directly via command? For example when the player talks to the lamberjack the player triggers a command like 
/shop category wood 
and only the category wood opens and not the category overwie like in /shop.

**Comment by Bounser** [comment]
yes, you can already use `/market category <category-identifier>` to access a certain category.
